script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -71,14 +71,14 @@ She sent me off with tears in the corners of her eyes, having already accepted t
 <Page20>
 That's what happened just 40 minutes ago.
 It was a turning point in my entire life.
-It was the beginning of a new life for me, and the end of Tohno Shiki's old life.
+It was the beginning of a new life for me, and the end of Tohno Shiki's old one.
 <Page21>
 The scenery outside is changing little by little.
 We must have passed through Yashirogi, a suburb across the river, and entered the urban area of Souya.
 <Page22>
 The train line emerges above ground again.
-The sensation of gently ascending a slope.
-My eyes, which had become accustomed to the artificial light, are stung by the sunlight as if I was being woken up.
+A sensation of gently ascending a slope.
+My eyes, which had become accustomed to the artificial light, are stung by the sunlight as if being woken up.
```

Comment:
Sounds maybe a little weird for one's eyes to be woken up instead of the person themselves

Reply:
I feel like Nasu was intentionally going for the eyes waking up, given that light is the subject as opposed to the noise of the train or whatever.

Reply:
Hrm, maybe @yk1512 can weigh in since he seems to be active
> 人工の明かりに慣れきった眼に、覚めるような陽が差しこんだ。

Reply:
'stung by sunlight' seems inaccurate to me. It reads as more of a gentle change from artifical light to 'real light'. 
I would maybe go as far as to change the sentence to something like 'My eyes, which had become accustomed to the artificial light, are rejuvenated by the sunlight.'